src/ipc/ipc4/handler.c

```diff
@@ -131,14 +131,14 @@ static inline const struct ipc4_pipeline_set_state_data *ipc4_get_pipeline_data(
 /*
  * Global IPC Operations.
  */
-static int ipc4_new_pipeline(struct ipc4_message_request *ipc4)
+__cold static int ipc4_new_pipeline(struct ipc4_message_request *ipc4)
```

Comment:
I assume the IPC irq TEXT is SRAM, but IRQ thread TEXT is DRAM and its stack is SRAM ?

Reply:
@lgirdwood this is just an example - it's marked "DNM" - I marked more or less random functions with `__cold` for a test. When we do merge this, we'll make careful choices - which one goes where